opsdroid/web.py

```diff
@@ -100,12 +174,18 @@ def base_url(self):
         """Return the base url of the opsdroid web server."""
         if self.config.get("base_url"):
             return self.config.get("base_url")
+        elif self.config.get("base-url"):
+            return self.config.get("base-url")
         protocol = "http" if self.get_ssl_context is None else "https"
         return f"{protocol}://{self.get_host}:{self.get_port}"
 
     async def start(self):
         """Start web servers."""
-        _LOGGER.info(_(f"Started web server on {self.base_url}"))
+        if self.command_center and not self.authorization_token:
```

Comment:
Seem that this is also triggered when the command centre is explicitly disabled and token is not set. This could confuse users.

Config to reproduce:
```:YAML
web:
  command-center:
    enabled: False
```